flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/GenericWriteAheadSinkTest.java

```diff
@@ -122,15 +123,15 @@ protected void verifyResultsWhenReScaling(
         }
 
         Collections.sort(sink.values);
-        Assert.assertArrayEquals(list.toArray(), sink.values.toArray());
+        assertThat(sink.values.toArray()).containsExactly(list.toArray());
     }
 
     @Test
```

Comment:
I know it's a bit out of scope but could we also move the `@Test` annotation underneath the JavaDoc here? ...probably in a separate hotfix commit

Reply:
done within this PR as https://github.com/apache/flink/pull/19916/commits/edcf9c5d16340caf114bae8af1a1ee9efff9afb5
please let me know if it is better to have another PR for this or not

Reply:
No, having this minor thing being done as part of this PR is good enough. :+1: